templates/engage/openstack-security-webinar.html

```diff
@@ -0,0 +1,54 @@
+{% extends "engage/base_engage.html" %}
+
+{% block title %}Private cloud security{% endblock %}
+
+{% block meta_description %}In regulated sectors, private cloud security needs to meet the highest requirements. This webinar explains how to deliver on these with an Openstack cloud.{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1Qg3sgDrvH9w_sgJpb2fp5lyOA5R-xhGZM3aKcyuQT68/edit?ts=5cbee163#{% endblock meta_copydoc %}
+
+{% block content %}
+
+{% include "engage/shared/_header.html" with title="Private cloud security" subtitle="How Charmed OpenStack is decreasing common security concerns" class="p-strip--dark p-takeover--openstack"%}
+
+<section class="p-strip is-shallow is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <p>
+        Telco, finance, insurance, healthcare, governments are among the regulated sectors, where security and in particular data privacy are fundamental to the design of a private cloud. Openstack is a popular solution to deploy secure private clouds. Implementing Openstack in regulated environments and matching stringent security requirement however can be operationally challenging and costly. 
+      </p>
+      <h4>
+        Watch this webinar to:
+      </h4>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">
+          Understand common practices to operate secure private clouds based on Openstack
+        </li>
+        <li class="p-list__item is-ticked">
+          Understand operational concerns associated with these practices 
+        </li>
+        <li class="p-list__item is-ticked">
+          Examine how automation can help address these concerns
+        </li>
+        <li class="p-list__item is-ticked">
+          Explore how Charmes Openstack can help deploy key security features:
+          <ul>
+            <li>
+              Offer encryption at rest as a service to private cloud users
+            </li>
+            <li>
+              Implement certificate lifecycle management to keep data secure in all scenarios
+            </li>
+          </ul>
+        </li>
+      </ul>
+      <p><a href="https://ubuntu.com/engage/openstack-security-webinar?utm_source=Takeover &utm_medium=Takeover&utm_campaign=CY19_DC_Openstack_WBN_OpenStackSecurity" class="p-button--positive">Register for webinar</a></p>
```

Comment:
This button needs to be taken out. The iframe webinar section (see my previous PR comments) should be added after the current one.